README.md

```diff
@@ -17,28 +17,37 @@ $ java -jar target/testSuite-1.0-SNAPSHOT-shaded.jar --rooturl http://localhost:
 
 ## Options
 * `rooturl` The repository base URL, e.g., `http://localhost:8080/rest/`
-* `admin-user` An user with admin access to the repository.
-* `admin-password` The admin user's password.
-* `user` The username to connect to the repository with
-* `password` The password to connect to the repository with
+* `root-controller-user-webid` An user with admin access to the repository.
+* `root-controller-user-password` The admin user's password.
+* `root-controller-user-auth-header-value` "Authorization" header value for a user with read, write, and control.  When present, this value will be added to the request effectively overriding Authenticator implementations, custom or default, found in the classpath.
+* `permissionless-user-webid` The username to connect to the repository with
+* `permissionless-user-password` The password to connect to the repository with
+* `permissionless-user-auth-header-value` "Authorization" header value for a user with no preset permissions.  When present, this value will be added to the request, effectively overriding Authenticator implementations, custom or default, found in the classpath.
 * `testngxml` (optional) The custom testng.xml configuration ([documentation](http://testng.org/doc/documentation-main.html#testng-xml))
   * See example [testng.xml](https://github.com/fcrepo/Fedora-API-Test-Suite/tree/master/src/main/resources/testng.xml)
 * `requirements` (optional) The requirement-levels of test to be run: ALL|MUST|SHOULD|MAY
   * Multiple levels can be provided, separated by ','
 * `config-file` (optional) A yaml configuration file containing the configuration parameters. See distributed `config.yml.dist`
 * `site-name` (optional) The above yaml file can contain multiple configurations, this chooses one. Defaults to "default"
 * `constraint-error-generator` (optional)  A file containing a SPARQL query that will trigger a constraint error. If no file is specified, the test suite will use a default constraint error generating test.
+* `auth-class` (optional) The class name of an implementation of the org.fcrepo.spec.testsuite.authn.Authenticator interface.
 
 ### Configuration file syntax
 The configuration file is Yaml and a simple structure. The first level groups a set of configuration parameters, these parameters are key value pairs with the keys being the above options.
 
+### Authenticators 
```

Comment:
This section seems out of place. Right above, we have `Configuration file syntax`. Right below, we have example configuration files.